templates/estate/index.html

```diff
@@ -1,293 +1,16 @@
 {% extends 'base.html' %}
```

Comment:
Mátt discarda allar breytingar í þessari skrá. Þetta yfirskrifar allt sem ég er búin að gera.